src/coreclr/src/gc/gc.cpp

```diff
@@ -16810,6 +17902,7 @@ int gc_heap::generation_to_condemn (int n_initial,
                     dprintf (BGC_TUNING_LOG, ("BTL[GTC]: trigger based on gen%d b: %Id",
                              (i),
                              get_new_allocation (i)));
+                    break;
```

Comment:
Is this a bug fix or a slight optimization?

Reply:
slight opt - if we are already doing max_gen, there's no point to check more 'cause that's the high gen we can condemn anyway.

Reply:
Ok.